modules/ocf_desktop/files/xsession/staff-logout

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+import datetime
+from subprocess import call
+from subprocess import PIPE
+from subprocess import Popen
+from time import sleep
+
+from ocflib.lab.hours import Day
+
+# delay before killing session in minutes
+delay = 45
+# extra delay if we will be closed delay minutes from now
+bonus = 30
```

Comment:
Well, people don't always know when they are going to collapse of exhaustion.....